helm/prometheus-rules/templates/platform/atlas/recording-rules/grafana-cloud.rules.yml

```diff
@@ -371,6 +371,10 @@ spec:
       record: aggregation:prometheus:memory_percentage
     - expr: sum(label_replace(container_memory_working_set_bytes{container='prometheus', namespace=~'.*-prometheus'}, "cluster_id", "$2", "pod", "(prometheus-)(.+)(-.+)")) by (cluster_id, cluster_type, customer, installation, pipeline, provider, region)
       record: aggregation:prometheus:memory_usage
+  - name: mimir.grafana-cloud.recording
+    rules:
+    - expr: sum(container_memory_working_set_bytes{namespace='mimir', cluster_type="management_cluster", container=~'.+'}) by (cluster_id, cluster_type, customer, installation, pipeline, provider, region)
```

Comment:
Maybe we want to add the number of scraped samples?

Reply:
oooh yes, the prometheus remotewrite config matches:
```
regex: (^aggregation:.+|prometheus_tsdb_head_series|prometheus_tsdb_head_samples_appended_total|^slo_.+)
```

So we probably want to have some replacement for `prometheus_tsdb_head_series` and `prometheus_tsdb_head_samples_appended_total` with mimir. Good point.

Reply:
Maybe we should make it a recording rules only?